src/runtime/wpt.ts

```diff
@@ -45,6 +45,7 @@ declare function async_test(f: (this: WptTestObject) => Promise<void>, name: str
           }
 
           this.step(() => {
+            // Unfortunately, it seems not possible to surface any logs for warn/skip.
```

Comment:
We still get them in standalone, right?

In wpt, could we console.warn them? Maybe behind a verbose flag?

Reply:
Yeah, standalone handles all these. Good idea to console.warn.

Reply:
I remembered WPT lints against console logging. Maybe we should just leave this as is, since people should hopefully be debugging on the standalone version anyway.

Reply:
(which also has the typescript source so source maps work)

Reply:
sgtm